bandit/blacklists/calls.py

```diff
@@ -569,4 +569,65 @@ def gen_blacklist():
         'attacks. Consider using tmpfile() instead.'
         ))
 
+    # updated rulesets starts here
+
+    sets.append(utils.build_conf_dict(
+        'yaml_load', 'B326',
+        ['yaml.load'],
+        """
+        Use of this deprecated deserialize method leads to
+        remote code execution. See CVE-2017-18342.
+        """
+        'Use yaml.safe_load instead.'
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'response_splitting', 'B327',
```

Comment:
This CVE was resolved already in Python 3.7, 3.8, and greater.  At this point, I think it's unnecessary for Bandit to flag that module.